_docs/_latest/configure/scanner/descriptions.md

```diff
@@ -166,3 +166,20 @@ than the max age.
 For examples of how to define scanner rules for your service account keys, see the
 [`service_account_key_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/rules/service_account_key_rules.yaml)
 file.
+
+
+## External Project Access Scanner
+
+Data exfiltration is a common threat in cyber security and in the Cloud.  The External Project Access Scanner mitigates this by identifying users who have access to projects where the project ancestor hierarchy does not include  whitelisted parent.  
```

Comment:
Limit each line to 80 characters.

Reply:
How about if we describe this more simply:
```
The External Project Access Scanner mitigates this by identifying users who have access to projects outside of your organization. 
```

Reply:
modified as suggested